radis/lbl/broadening.py

```diff
@@ -328,35 +328,40 @@ def pressure_broadening_HWHM(
 
     diluent_molecules = diluent.keys()
 
-    # check if gamma_diluent and n_diluent exists or not
-    try:
-        gamma_lb = 0
-        for diluent_molecule, diluent_mole_fraction in diluent.items():
-            if diluent_molecule != "air":
-                gamma_lb += (
-                    (Tref / Tgas)
-                    ** diluent_broadening_coeff["n_" + diluent_molecule.lower()]
-                ) * (
-                    diluent_broadening_coeff["gamma_" + diluent_molecule.lower()]
-                    * pressure_atm
-                    * diluent_mole_fraction
+    # check if gamma_diluent and n_diluent exists or not and in case they are not present use broadening coefficient of air instead
+    gamma_lb = 0
+    for diluent_molecule, diluent_mole_fraction in diluent.items():
+            try:
+                    gamma_lb += (
+                        (Tref / Tgas)
+                        ** diluent_broadening_coeff["n_" + diluent_molecule.lower()]
+                    ) * (
+                        diluent_broadening_coeff["gamma_" + diluent_molecule.lower()]
+                        * pressure_atm
+                        * diluent_mole_fraction
+                    )
+            except:
```

Comment:
please put a class of error to catch by "except", such as `except ValueError:`. This is to avoid that other errors are treated by this error handling

Reply:
done